paasta_tools/check_chronos_jobs.py

```diff
@@ -90,6 +90,23 @@ def sensu_event_for_last_run_state(state):
         return pysensu_yelp.Status.OK
 
 
+def filter_most_recent_jobs_for_service(jobs):
+    most_recent = {}
+    for job in jobs:
+        service, instance = chronos_tools.decompose_job_id(job['name'])
+        if instance not in most_recent:
```

Comment:
Got it, thanks for the feedback. I'll ship this and file a ticket for that refactor/cleanup for later.